src/components/walletconnect/ProposalForm/bridges.ts

```diff
@@ -0,0 +1,43 @@
+const DangerousBridges = [
```

Comment:
Naming-wise, there's nothing dangerous/risky about these bridges per se, it's just that they aren't smart contract account-friendly. Please find more specific names for the two arrays. And please move these lists (not the two functions below) to `src/components/walletconnect/constants.ts`.